lib/providers/services/eos_service.dart

```diff
@@ -170,7 +173,10 @@ class EosService {
   }
 
   Future<dynamic> acceptInvite(
-      {String accountName, String publicKey, String inviteSecret, String nickname}) async {
```

Comment:
Is this formatting from automated settings in VS code? If so can you check them in so we can all use the same?